app/hotels/src/singleHotel/HotelDetailScreen.js

```diff
@@ -112,31 +120,43 @@ export class HotelDetailScreen extends React.Component<Props, State> {
     }
 
     return (
-      <React.Fragment>
-        <ScrollView contentContainerStyle={styles.container}>
-          <LayoutSingleColumn>
-            <AdaptableLayout
-              renderOnWide={<View style={styles.marginView} />}
+      <SafeAreaView style={styles.safeArea} forceInset={{ top: 'never' }}>
+        <View style={styles.safeArea}>
+          <ScrollView contentContainerStyle={styles.container}>
+            <LayoutSingleColumn>
+              <AdaptableLayout
+                renderOnWide={<View style={styles.marginView} />}
+              />
+              <Header hotel={availableHotel.hotel} />
+              <HotelInformation hotel={availableHotel.hotel} />
+              <RoomList
+                data={availableHotel.availableRooms}
+                select={this.selectRoom}
+                deselect={this.deselectRoom}
+                selected={selected}
+              />
+              <BrandLabel />
+            </LayoutSingleColumn>
+          </ScrollView>
+          <View style={styles.buttonContainer}>
+            <RoomSummary
```

Comment:
Maybe the props should be renamed `guestNumber`/`guestQuantity` and `roomNumber`/`roomQuantity` to make it more descriptive of what it is.

Reply:
I will go for guestCount/roomCount

Reply:
That's much better than my suggestions :D